Gruntfile.js

```diff
@@ -70,15 +83,32 @@ module.exports = function(grunt) {
         push: false,
       }
     },
+
+    clean: {
+      build: ['build/*.js', 'build/*.map'],
```

Comment:
Just nukes the files before the compress + copy. I mainly wanted this to ensure the compression step builds something and doesn't copy over something it shouldn't. I'll add something to nuke the dist dir too. That way if something is not generated we'll know.